src/main/java/pl/zimi/zadanka/Zad4.java

```diff
@@ -1,8 +1,29 @@
 package pl.zimi.zadanka;
+
+import java.util.ArrayList;
+import java.util.List;
+
 /*
 source
```

Comment:
Pomijając wszystkie uwagi Twoje rozwiązanie jest nieprawidłowe
widziałeś przykład z tej strony?

Input: s = "the sky is blue"
Output: "blue is sky the"

masz odwrócić nie litery w stringu a wyrazy w zdaniu